starter-code/scripts/models/article.js

```diff
@@ -49,16 +49,31 @@ Article.fetchAll = function() {
     /* When our data is already in localStorage:
     1. We can process and load it,
     2. Then we can render the index page.  */
+    var ipsum = JSON.parse(localStorage.getItem('hackerIpsum'));
```

Comment:
You are saving your parsed article data to a variable, but you're not doing anything with it. We need to use the Article.load method and pass it our data in order to populate the articles array and then render the page.